src/binaryen_stubs_expressions.c

```diff
@@ -148,9 +156,13 @@ caml_binaryen_return_call_indirect(value _module, value _target, value _params,
   }
   BinaryenType paramsty = BinaryenType_val(_paramsty);
   BinaryenType retty = BinaryenType_val(_retty);
-  BinaryenExpressionRef exp = BinaryenReturnCallIndirect(module, target, params, paramsLen, paramsty, retty);
+  BinaryenExpressionRef exp = BinaryenReturnCallIndirect(module, table, target, params, paramsLen, paramsty, retty);
   CAMLreturn(alloc_BinaryenExpressionRef(exp));
 }
+CAMLprim value
+caml_binaryen_return_call_indirect__bytecode(value * argv) {
+  return caml_binaryen_return_call_indirect(argv[0], argv[1], argv[2], argv[3], argv[4], argv[5]);
+}
```

Comment:
Why did you need to add this?

Reply:
If you want to call a C function from OCaml that's got more than 5 arguments, you need to provide two functions—one that does the actual work and another which calls it with the actual arguments. It's probably because OCaml's calling convention only creates functions of up to five arguments or something like that. But you'll see throughout the rest of the stubs that this is done for anything that takes more than 5.

Reply:
Thanks for the explanation!